test/compilable/cppmangle3.d

```diff
@@ -1,4 +1,6 @@
+module cppmangle3;
 
+import std.traits : fullyQualifiedName;
```

Comment:
Using Phobos is not allowed in the test suite.  The test below is sufficient.

Reply:
In order to test the name, you could also add a test to `fail_compilation` which checks the error message.

Reply:
That's a good idea.

Reply:
Yah I couldn't think of one either, the only one that I could think of was removed. The symbol clashing lol.

Reply:
There are heaps of error messages that do emit the symbol name. I ran into a bunch in short order... but now i've got nothing! ;)